hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/BlockStreamAccess.java

```diff
@@ -202,7 +213,7 @@ private Block blockFrom(@NonNull final Path path) {
                 return Block.PROTOBUF.parse(Bytes.wrap(Files.readAllBytes(path)));
             }
         } catch (IOException | ParseException e) {
-            throw new RuntimeException(e);
+            throw new RuntimeException("Failed reading block @ " + path, e);
```

Comment:
nit: may be worth while now to normalize and sanitize the input path we are reading from